Combined PR review

cristinamullin opened the pull request:
This PR combines edits made in these 3 PRs for team review: 

1. Creates two new functions: TADA_CreateUseAURef and TADA_CreateWaterUseRef: https://github.com/USEPA/EPATADA/pull/623

- Assigns ATTAINS Use Names to each AU in a user's WQP/TADA dataframe
- TADA_CreateWaterUseRef is a helper function to help assign ATTAINS Use to new/updated AUs

2. New Module 2/Geospatial Workflow: https://github.com/USEPA/EPATADA/pull/629

- A functional draft of TADA_CreateAUMLRef
- An example Module 2 workflow (see vignette called ExampleMod2Workflow.rmd)

4. Bug fixes & maintenance: https://github.com/USEPA/EPATADA/pull/624

- Incorporate recent updates into vignettes
- Add tests to avoid NA result and unit issues in the future
- Update TN and TP harmonization and summation reference tables: https://github.com/USEPA/EPATADA/issues/319
- Update TADA_ConvertSpecialChars clean = T option
- Remove TADA_AutoFilter and add functionality to TADA_RetainRequired
- Fix bug in TADA_ConvertSpecialChars where we were getting new columns with TADA.TADA.
- Fix bug in TADA_CalculateTotalNP where duplicate rows were being added when TP or TN already were in the dataset for a given day/location
- Add metadata to new rows created by TADA_CalculateTotalNP. Addresses issue: https://github.com/USEPA/EPATADA/issues/588
- Adds curl as as dependency. Addresses: https://github.com/USEPA/EPATADA/issues/627
- Updated TADA_AggregateMeasurements and TADA_RunKeyFlagFunctions
- Fixes NA bug in TADA_CalculateTotalNP and improve flags

wokenny13 committed:
Incorporate Mod 3 spatial ref code back

wokenny13 committed:
Reflects Most Recent Updates from TADA_CreateParamRef and TADA_CreateUseParamRef PR

wokenny13 committed:
Update CriteriaMethods.R

Excel conditional formatting update

wokenny13 committed:
Update CriteriaMethods.R

handle empty column type for joining data format

wokenny13 committed:
Update ATTAINSRefTables.R

Manual additional of additional alias between ATTAINS ParameterName and WQP CharacteristicName

wokenny13 committed:
Update ATTAINSCrosswalks.R

Changed to "Not Applicable for Analysis" as the drop down value for TADA_CreateParamRef

wokenny13 committed:
Update ATTAINSCrosswalks.R

Some code styling changes and arranged drop down by ascending.

Do we want the "Not applicable for Analysis" option to show up in the top or bottom in Excel?

We mentioned top but it seems like clicking on the drop down will show bottom entries first

wokenny13 committed:
Merge branch 'develop' into Create-Module-3-AURef

wokenny13 committed:
Update GeospatialFunctions.R

Changed offset to 100 for ATTAINS geospatial webservices

wokenny13 committed:
CriteriaSummary and CriteriaTable Drafts

wokenny13 committed:
Spatial Ref table AU Bug Fix

Renamed to consistent column names between helper function and spatial crosswalk with AU spatial summary.

wokenny13 committed:
Update CriteriaMethods.R

Added Char, spec, frac to CriteriaMethods.

Reformat excel file

wokenny13 committed:
CriteriaMethod

Changed column name to ATTAINS.WaterType throughout crosswalk tables.

Edited format of excel spreadsheet outputs.

wokenny13 committed:
Update CriteriaMethodology

Edit to required columns.

DepthProfile fix incorporated

wokenny13 committed:
Mod 3 ref table formatting

cristinamullin committed:
Update DepthProfile.R

wokenny13 committed:
Merge branch 'develop' into Create-Module-3-AURef

wokenny13 committed:
Moved two spatial functions from Mod 3 to Mod 2 and Reconsideration of workflow

Reordered workflow of functions and changed function name

New order:
TADA_CreateWaterUseRef -> TADA_CreateUseAURef

Users should run these functions as part of Module 2 now.

These functions aim to help with assigning Uses to AU once sites have been joined to AUs.

wokenny13 committed:
Update CriteriaMethods.R

wokenny13 committed:
Update CriteriaMethods.R

wokenny13 committed:
Update CriteriaMethods.R

hillarymarler committed:
minor fix to adding data links, vignette updates

hillarymarler committed:
Update r5AUMatchingDemo.Rmd

wokenny13 committed:
Update CriteriaMethods.R

wokenny13 committed:
Auto Generate Criteria Methods Option

wokenny13 committed:
Use to AU Ref Updates

wokenny13 committed:
Join Use to AUs

This PR focuses on assigning ATTAINS Use Name to each AU.

There are two functions in this PR: TADA_CreateUseAURef and TADA_CreateWaterUseRef

TADA_CreateWaterUseRef is a helper function to help assign ATTAINS Use to new/updated AUs.

This function excludes other Mod 3 functions for the time being.

hillarymarler committed:
Merge branch 'develop' into Use-to-AU-Reference-Table

wokenny13 committed:
function input variable name changed

Used a sample data input for testinf and forgot to change back to sitesAURef

wokenny13 committed:
Fix Check Issues and Other Maintainence

Added global variables to utilities

Fixed Function documentation in CreateWaterUseRef

Ran styler and word list maintainence

wokenny13 committed:
Run document

Run document.

Look at the new function example for inputting a custom useAURef. Do we want this as the output where the 5 new AUs are assigned to a no use name but show as excluded?

The goal would be to identify any combinations that are not captured in the user supply useAURef table.

wokenny13 committed:
Exclude documents

Exclude CriteriaMethods.R from this PR. Not meant to be a part of this PR and will be a separate item.

hillarymarler committed:
Update ATTAINSCrosswalks.R

Add ATTAINS.watertype to output from TADA_GetATTAINSAUSiteCrosswalk

hillarymarler committed:
Merge branch 'Use-to-AU-Reference-Table' of https://github.com/USEPA/EPATADA into Use-to-AU-Reference-Table

hillarymarler committed:
Update ATTAINSCrosswalks.R

wokenny13 committed:
documentation updates

Removed missing function link, update global variables, added a missing param arg input.

hillarymarler committed:
Option for streamlining TADA_GetATTAINSParamUseOrgRef

relies on using the national download rather than looping through a web request to each org. Currently, it uses only the latest assessment cycle (similar to the old version of this function). If you wanted it to include params from all cycles (in ATTAINS) the "latest.assessments" section could be removed and the latest.params df could be generated from the original nat.assessments

hillarymarler committed:
Merge branch 'Use-to-AU-Reference-Table' of https://github.com/USEPA/EPATADA into Use-to-AU-Reference-Table

hillarymarler committed:
bug fix to retain water type

wokenny13 committed:
Removed unused function param

in TADA_CreateWaterUseRef

cristinamullin committed:
Merge branch 'develop' into Use-to-AU-Reference-Table

cristinamullin committed:
add new TADA_ConvertSpecialChars clean = T option

cristinamullin committed:
fix TN and TP harmonization

add unit to grouping cols for TADA_AggregateMeasurements, fix TN and TP bug, add combos to harmonization table, update vignette

cristinamullin committed:
Update TADAModule1.Rmd

remove intermediate variables and update scatterplot examples

cristinamullin committed:
Update TADAModule1.Rmd

cristinamullin committed:
removes TADA_AutoFilter and adds functionality to TADA_RetainRequired

Combine TADA_AutoFilter with TADA_RetainRequired and delete TADA_AutoFilter

cristinamullin committed:
update vignettes

incorporate changes made to TADA_FindQCActivities, TADA_RetainRequired, and TADA_AutoFilter

cristinamullin committed:
Update TADA_RetainRequired.Rd

cristinamullin committed:
Update TADAModule1.Rmd

cristinamullin committed:
update refs

cristinamullin committed:
update example data

cristinamullin committed:
add missing rows to NP/TN summation key, add metadata to output

wokenny13 committed:
Minor text fix

Removed comment

cristinamullin committed:
Update RequiredCols.R

cristinamullin committed:
add tests

cristinamullin committed:
only run if not run previously

cristinamullin committed:
define flag col and add tests

add tests and examples for TADA_ConvertSpecialChars and TADA_CalculateTotalNP

hillarymarler committed:
function update to attains crosswalk

work on improving updates to MLID and URL

cristinamullin committed:
revert column additions

hillarymarler committed:
Update ATTAINSCrosswalks.R

hillarymarler committed:
Update ATTAINSCrosswalks.R

cristinamullin committed:
add curl as dependency

hillarymarler committed:
geospatial updates

hillarymarler committed:
updates

hillarymarler committed:
Update GeospatialFunctions.R

cristinamullin committed:
update TADA_CalculateTotalNP

add dataRetrieval depends to EPATADA depends, improve print messages, add example to TADA_CalculateTotalNP, address QC and non-numeric results within TADA_CalculateTotalNP, add TADA_CalculateTotalNP tests

hillarymarler committed:
Update r5AUMatchingDemo.Rmd

hillarymarler committed:
Update r5AUMatchingDemo.Rmd

hillarymarler committed:
Update r5AUMatchingDemo.Rmd

hillarymarler committed:
Merge branch 'R5_AUMatchDemo' into compatible_au_cols

cristinamullin committed:
update TADA_CreateUseAURef documentation

cristinamullin committed:
add details to TADA_CreateUseAURef documentation

hillarymarler committed:
Update GeospatialFunctions.R

update col names for ATTAINS function - start w/ TADA_GetATTAINSByAUID

hillarymarler committed:
update TADA_MakeSpatial

switch to using TADA prefixed lat/lon cols

hillarymarler committed:
Update GeospatialFunctions.R

update attains_names in TADA_GetATTAINS, additional catchment work for getting data by AUID

hillarymarler committed:
remove au_ref from TADA_GetATTAINS

hillarymarler committed:
troubleshooting for creating TADA_with

cristinamullin committed:
organize functions

move functions to more applicable .R files

cristinamullin committed:
update docs

wokenny13 committed:
UseAURef excel return option fixes and empty dataframe creation

sitesAURef arg input changed to MLAURef name change.

if arg inputs are all blank, return empty dataframe.

will only return an empty dataframe in R, not in excel. Would users find the excel export useful if they want an empty dataframe?

For the excel output, I made a separate excel file created for this function.

hillarymarler committed:
column renaming updates

hillarymarler committed:
started combined function draft

start of draft function, minor changes to col names in additional functions

cristinamullin committed:
Update Transformations.R

cristinamullin committed:
update TADA_AggregateMeasurements and TADA_RunKeyFlagFunctions

the default should be clean = FALSE for TADA_AggregateMeasurements and TADA_RunKeyFlagFunctions, add better messages for flag functions

wokenny13 committed:
Create empty dataframe option for ParamRef and UseParamRef

Also changed MLAURef to AUMLRef

hillarymarler committed:
updates to TADA_GetATTAINS and new combination function

hillarymarler committed:
col name corrections

hillarymarler committed:
adding ref source col

hillarymarler committed:
add color group option to TADA_ViewATTAINS

not yet tested

hillarymarler committed:
small bug fixes

hillarymarler committed:
work on adding additional colors to TADA_ViewATTAINS

additional colors on map not working yet

hillarymarler committed:
color updates for view Attains

cristinamullin committed:
update TADA_CalculateTotalNP

add more TN/TP combinations, update TADA_CalculateTotalNP documentation, add tests

hillarymarler committed:
updates to ex workflow

hillarymarler committed:
fixed row binding bug

hillarymarler committed:
functional draft of workflow

hillarymarler committed:
move new function to geospatial and documentation

hillarymarler committed:
add function specific messages to TADA_CreateAUMLRef

cristinamullin committed:
update TADA_CalculateTotalNP

add additional matches, handle aggregated values, handle non-numeric results

hillarymarler committed:
updates to makespatial and findnearby

to be consistent with using the TADA prefixed lat and lon cols in functions

hillarymarler committed:
fix color for TADA_GetATTAINS only in ViewATTAINS

cristinamullin committed:
fix NA bug in TADA_CalculateTotalNP

hillarymarler committed:
Update GeospatialFunctions.R

hillarymarler committed:
update refs to col names

hillarymarler committed:
rename function to retrieve cw from ATTAINS

hillarymarler committed:
function renaming and doc updates

hillarymarler committed:
rename TADA_UpdateATTAINSAUMLCrosswalk

hillarymarler committed:
Update TADAWaterSciConWorkshopDemo.Rmd

hillarymarler committed:
rename TADA_GetATTAINS to TADA_CreateATTAINSAUMLCrosswalk

hillarymarler committed:
documentation updates

hillarymarler committed:
add batch_upload option to TADA_GetATTAINSAUMLCrosswalk

hillarymarler committed:
add batch_upload param

add batch_upload param to functions which generate AU/ML crosswalks

cristinamullin committed:
remove duplicated info, improve TADA_CalculateTotalNP flags

hillarymarler committed:
batch_upload updates

small fixes to make batch_upload work in/for multiple functions

hillarymarler committed:
fixes for vignette errors and update to TADA_DataRetrieval

updated TADA_DataRetrieval to reflect TADA_MakeSpatial running off TADA-prefixed cols now. Update branch refs for vigenttes

hillarymarler committed:
Update TADACybertown2025.Rmd

updated branch ref

hillarymarler committed:
vignette updates

hillarymarler committed:
removed unnecessary example lines from CreateAUMLRef

cristinamullin committed:
fix TADA_CalculateTotalNP and TADA_IDCensoredData logic and create tests

fix tests and add more, update logic

hillarymarler committed:
test data updates for geospatial tests

hillarymarler committed:
geospatial test updates

hillarymarler committed:
Update TADA_CreateAUMLRef.Rd

hillarymarler committed:
Delete r5AUMatchingDemo.Rmd

removed R5 matching demo vignette from this PR (need additional updates to reflect function changes)

hillarymarler committed:
Update TADAModule1.Rmd

changed group for scatterplot example (can probably be changed back after CM's updates to nutrient functions)

hillarymarler committed:
Update ExampleMod2Workflow.Rmd

removed duplicate chunk

hillarymarler committed:
Update GeospatialFunctions.R

documentation update

hillarymarler committed:
documentation updates for geospatial functions

cristinamullin committed:
update tests

cristinamullin committed:
Merge branch 'compatible_au_cols' into combined-review

cristinamullin committed:
document

cristinamullin committed:
Merge branch 'Use-to-AU-Reference-Table' into combined-review

hillarymarler commented on R/GeospatialFunctions.R:
```diff
@@ -2166,3 +2626,912 @@ TADA_ViewATTAINS <- function(.data) {
+
+
+
+#' Identify and group nearby monitoring locations (UNDER ACTIVE DEVELOPMENT)
```
Should we update this function for the option to find nearby sites within the same assessment unit? That would further reduce the size of the geospatial queries needed and would make sense in our proposed workflow if we think AUs should be assigned before deciding to analyze multiple sites as one.

cristinamullin replied:
That approach makes sense to me!

hillarymarler replied:
Cool - I will add an issue for this and will work on it after we're done troubleshooting this PR.

hillarymarler commented on DESCRIPTION:
```diff
@@ -58,7 +58,7 @@ Imports:
     stringi,
```
With increasing size/dependencies, should we revisit the conversation about splitting EPATADA into multiple packages?

cristinamullin replied:
Yes, let's discuss the pros & cons. Adding this to the TADA Team meeting agenda.

hillarymarler commented on NAMESPACE:
```diff
@@ -13,12 +12,16 @@ export(TADA_ColorPalette)
 export(TADA_ConvertResultUnits)
 export(TADA_ConvertSpecialChars)
+export(TADA_CreateATTAINSAUMLCrosswalk)
+export(TADA_CreateAUMLRef)
```
Most of the other geospatial function updates I made used function names that included "Crosswalk" at the end, not "Ref". Should I change this one to "Crosswalk" as well for consistency. That would leave the "Ref" functions more specifically referring to the functions Kenny wrote for the assessment/analysis steps.

cristinamullin replied:
I agree "Crosswalk" makes sense for this one for consistency.

hillarymarler replied:
I will make that change now.

hillarymarler committed:
Update ATTAINSCrosswalks.R

changeTADA_UpdateATTAINSAUMLCrosswalk was not reflected in combined review PR, added all of these changes back

hillarymarler committed:
Merge branch 'combined-review' of https://github.com/USEPA/EPATADA into combined-review

wokenny13 commented on vignettes/ExampleMod2Workflow.Rmd:
```diff
@@ -0,0 +1,179 @@
+    "MDEQ_WQ_WQX-C04KNDYC04", "MDEQ_WQ_WQX-C04KNDYC54")) %>%
+  dplyr::rename(AssessmentUnitIdentifier = ATTAINS.AssessmentUnitIdentifier,
+                MonitoringLocationIdentifier = ATTAINS.MonitoringLocationIdentifier)
+
```
@hillarymarler are you still planning to include ATTAINS.WaterType as a column in TADA_GetATTAINSAUMLCrosswalk() that you've made in PR #623?

hillarymarler replied:
Yes - it is now included in this combined PR

hillarymarler committed:
Update ATTAINSCrosswalks.R

cristinamullin committed:
Delete .RDataTmp1

hillarymarler committed:
bug fix for TADA_UpdateATTAINSAUMLCrosswalk

hillarymarler committed:
Merge branch 'combined-review' of https://github.com/USEPA/EPATADA into combined-review

hillarymarler commented on R/ATTAINSCrosswalks.R:
```diff
@@ -207,38 +240,41 @@ TADA_GetATTAINSAUSiteCrosswalk <- function(org_id = NULL) {
-                                                    crosswalk = NULL,
-                                                    attains_replace = FALSE,
-                                                    wqp_data_links = "add") {
+TADA_UpdateATTAINSAUMLCrosswalk <- function(org_id = NULL,
```
Added changes back in which introduce these additional arguments (make function more efficient/flexible for use in other functions or workflow)

wokenny13 committed:
Update TADA_CreateUseAURef R documentation examples

reflects the name change to TADA_UpdateATTAINSAUMLCrosswalk()

wokenny13 committed:
Update TADA_CreateUseAURef.Rd

hillarymarler commented on R/ATTAINSCrosswalks.R:
```diff
@@ -596,7 +831,7 @@ TADA_UpdateMonitoringLocationsInATTAINS <- function(org_id = NULL,
 #' )
 #' # First, run key flag functions and harmonize synonyms across
```
I ran these examples and they worked well.

hillarymarler committed:
Update ATTAINSCrosswalks.R

updates to make KW and HM updates work together for creating Mod3 Ref dfs

hillarymarler committed:
Merge branch 'combined-review' of https://github.com/USEPA/EPATADA into combined-review

hillarymarler committed:
Update ATTAINSCrosswalks.R

hillarymarler committed:
CreateATTAINSAUMLCrosswalk updates

hillarymarler committed:
Update ATTAINSCrosswalks.R

Updates to TADA_CreateUseAURef, TADA_GetATTAINSAUMLCrosswalk, and TADA_UpdateATTAINSAUMLCrosswalk to facilitate the inclusion of ATTAINS.OrganizationIdentifier

hillarymarler committed:
updates to TADA_CreateWaterUseRef

column name updates and some suggested changes

hillarymarler commented on R/ATTAINSRefTables.R:
I'm not very clear on what belongs in ATTAINSCrosswalks.R vs ATTAINSRefTables. Should we discuss this as a group? Or all the functions in the correct place already?

cristinamullin replied:
I was thinking the reference tables are just ATTAINS domain tables/data we reference and bring in (like the WQX refs/domains), while the crosswalks are generated by TADA or users (integrating ATTAINS and WQP).

hillarymarler committed:
Update ATTAINSRefTables.R

make rExpertQuery functions run quietly

hillarymarler committed:
Update ATTAINSRefTables.R

make ExpertQuery functions run quietly

hillarymarler commented on R/DepthProfile.R:
```diff
@@ -89,8 +89,9 @@
 #'
-TADA_FlagDepthCategory <- function(.data, bycategory = "no", bottomvalue = 2, surfacevalue = 2,
-                                   dailyagg = "none", aggregatedonly = FALSE, clean = FALSE) {
+TADA_FlagDepthCategory <- function(.data, bycategory = "no", bottomvalue = 2, 
```
I like all the printed message edits in this function.

hillarymarler committed:
Update TADA_GetATTAINSAUMLCrosswalk.Rd

wokenny13 committed:
Update Transformations.R

@cristinamullin I took a stab on fixing the error in TADA_CalculateTotalNP.

Please review the edits to see if they make sense.

For the fix I noticed for
TADA_CalculateTotalNP that it seems to default to clean = FALSE in TADA_AggregateMeasurements, which leads to a TADA.ResultValueAggregation.Flag value being "Used in ..." rather than "Considered in ..." so made this change in line 494 as well as handle 'max', 'min', 'mean' rather than just max.

wokenny13 committed:
Update Transformations.R

handling of min, max or mean in TADA.ResultValueAggregation.Flag for daily_agg arg input.

cristinamullin commented on R/Transformations.R:
```diff
@@ -491,8 +491,10 @@ TADA_CalculateTotalNP <- function(.data,
   dat_addback <- dat[(dat$TADA.ResultValueAggregation.Flag %in%
-                        "Considered in max aggregation function but not selected"), ]
-  dat_addback$TADA.NutrientSummation.Flag <- "Not used to calculate Total N or P."
+                        c(paste0("Used in ", daily_agg, " aggregation function but not selected"))), ]
```
Good catch. I am updating this in both functions for consistency to:

**Considered in**

cristinamullin committed:
update flag language in TADA_AggregateMeasurements

cristinamullin committed:
update refs and example data

cristinamullin committed:
do not evaluate Mod 1 vignette

cristinamullin committed:
update example

hillarymarler committed:
change the name of TADA_CreateAUMLRef to TADA_CreateAUMLCrosswalk

hillarymarler committed:
update branch ref for WaterSciCon vignette

cristinamullin committed:
Update test-CensoredDataSuite.R

cristinamullin committed:
Update ExampleMod2Workflow.Rmd

cristinamullin committed:
Update ExampleMod2Workflow.Rmd

wokenny13 committed:
minor updates to TADA_GetATTAINSAUMLCrosswalk

reorder of columns considerations.

fix bug when batch_upload = TRUE

documentation text edits to reflect updates.

Additional example and other documentation example error fixes.

wokenny13 committed:
Merge branch 'combined-review' of https://github.com/USEPA/EPATADA into combined-review

wokenny13 committed:
Text edits

cristinamullin committed:
Update TADA_CalculateTotalNP

handle empty dfs

cristinamullin committed:
specify col types for TADAPriorityCharConvertRef.csv

TADAPriorityCharConvertRef.csv, update tests

wokenny13 committed:
update TADA_UpdateATTAINSAUMLCrosswalk

adding ATTAINS.OrganizationIdentifier column in a user supplied crosswalk.

fixed handling of bug with TADA_CreateUseAURef due to mismatch column names

wokenny13 committed:
Update TADA_CreateUseAURef

Handling of duplicate row bug

wokenny13 committed:
Update R documentation function name changes error in checks

wokenny13 committed:
Update TADA_CreateUseAURef.Rd

cristinamullin committed:
update documentation and tests

cristinamullin committed:
address some check notes

wokenny13 committed:
Remove, reorder and additional error handling bug fix

changed checking of column names in user supplied crosswalk from 'any' to 'all' to determine if user has included all required columns in their data frame input.

Return desired column in ATTAINS batch upload or TADA recommended workflow format.

Other minor text changes

cristinamullin committed:
always update censored data flag if simple censored data is run

wokenny13 committed:
address some over 100 line checks

wokenny13 committed:
run document

wokenny13 committed:
Merge branch 'combined-review' of https://github.com/USEPA/EPATADA into combined-review

cristinamullin committed:
handle empty dfs and update TADA_CalculateTotalNP

cristinamullin committed:
Update test-Utilities.R

cristinamullin committed:
update censored checks

cristinamullin committed:
improve TADA_ConvertSpecialChars

cristinamullin committed:
Update pH harmonization test

cristinamullin committed:
Update TADA_ConvertSpecialChars

handle Non-ASCII Character(s) when clean = FALSE in TADA_ConvertSpecialChars

hillarymarler committed:
fix undocumented argument issues

hillarymarler committed:
Merge branch 'combined-review' of https://github.com/USEPA/EPATADA into combined-review

cristinamullin committed:
up ph harmonization test

cristinamullin committed:
skip tests if random data is null

cristinamullin committed:
Update test-Utilities.R

cristinamullin committed:
update examples

cristinamullin committed:
update refs and example data

cristinamullin committed:
add example to TADA_CreateATTAINSAUMLCrosswalk

cristinamullin committed:
Update test-CensoredDataSuite.R

cristinamullin committed:
update examples

cristinamullin committed:
Update test-Utilities.R

cristinamullin committed:
update tests

cristinamullin committed:
Update test-Utilities.R

cristinamullin committed:
update tests

cristinamullin committed:
TADA_CalculateTotalNP update

handle NULL dfs

cristinamullin committed:
small improvements

TADA_CreateAUMLCrosswalk, TADA_IDCensoredData, Mod 2 vignette

wokenny13 committed:
Fix bug in ExampleMod2Workflow vignette

wokenny13 committed:
Update R documentation example

ATTAINS.WaterType was missing in the example.

Updated some text.

wokenny13 committed:
set eval = T in vignettes

wokenny13 committed:
set eval = T in vignettes

hillarymarler committed:
Update ExampleMod2Workflow.Rmd

set eval - TRUE

cristinamullin committed:
add volunteer monitoring vignette

cristinamullin committed:
Merge branch 'combined-review' of https://github.com/USEPA/EPATADA into combined-review

wokenny13 committed:
Merge branch 'combined-review' of https://github.com/USEPA/EPATADA into combined-review

wokenny13 committed:
eval = T

cristinamullin committed:
Merge branch 'combined-review' of https://github.com/USEPA/EPATADA into combined-review

cristinamullin committed:
Update Participatory-Science-Water-Projects-in-WQX.Rmd

cristinamullin committed:
Update Participatory-Science-Water-Projects-in-WQX.Rmd

cristinamullin committed:
update vignettes

hillarymarler committed:
Update TADACybertown2025.Rmd

update branch ref for knitting vignette

cristinamullin committed:
Update TADAModule1.Rmd

don't run multi org dups

cristinamullin committed:
Update TADACybertown2025.Rmd
